core/rawdb/database_quorum.go

```diff
@@ -27,9 +27,11 @@ import (
 )
 
 var (
-	privateRootPrefix           = []byte("P")
-	privateBloomPrefix          = []byte("Pb")
-	quorumEIP155ActivatedPrefix = []byte("quorum155active")
+	privateRootPrefix                      = []byte("P")
+	privateBloomPrefix                     = []byte("Pb")
+	quorumEIP155ActivatedPrefix            = []byte("quorum155active")
+	privateRootToPrivacyMetadataRootPrefix = []byte("PSR2PMDR")
+	emptyRoot                              = common.HexToHash("56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421")
```

Comment:
it may be worth to comment on this value:
```
// emptyRoot is the known root hash of an empty trie. Duplicate from `trie/trie.go#emptyRoot`
```

Reply:
added the suggested comment